modules/serverless-gclb/main.tf

```diff
@@ -149,45 +149,3 @@ locals {
   )
 }
 
-resource "google_monitoring_alert_policy" "abnormal-gclb-access" {
```

Comment:
should we keep this one, we dont have that many instances of serverless-gclb, it wont save much?

Reply:
This is the GCLB used in dev envs right? I don't think keeping it has the effect on non-dev/test envs

Reply:
Agree with Nghia that this isn't what we use in prod, so little benefit to keep.  If we want it, we should add it to the mono copy.

Reply:
ack, remove it